client/src/Booking.jsx

```diff
@@ -47,6 +47,7 @@ class Booking extends React.Component {
 
   componentDidMount() {
     this.init();
+    this.book();
     this.bookingTotal(); //just right now for testing endpt 3
   }
 
```

Comment:
if all you care about is `res.data` from your line 56, then consider destructuring up front:

```node.js
.then(({ data })) => {
  const {
    name,
    campId,
    price_per_night,
    how_far_out,
    cleaning_fee,
    max_guests
    instant_book
  } = data


  this.setState({
        name,
        campId,
        price_per_night,
        how_far_out,
        weeknight_discount,
        instant_book: instant_book ? true : false,
        cleaning_fee,
        max_guests,
  })
}) 
```